2.6/Dockerfile

```diff
@@ -1,10 +1,19 @@
 FROM centos:centos7
-MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
+LABEL MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
```

Comment:
Thanks for review.

I saw this in Fedora Container Guidelines (https://fedoraproject.org/wiki/Container:Guidelines), so I thought that this is right way so I simply changed it in this way. Now I see that in same guidelines in different example there is also your proposed syntax :-)

@hhorak How to deal with specifying maintainers in sclorg/ images? Are there any internal guidelines about similar requirements?